app/dashboard/admin.py

```diff
@@ -470,11 +470,19 @@ class PollsAdmin(admin.ModelAdmin):
 
 
 class QuestionsAdmin(admin.ModelAdmin):
-    list_display = ['id', 'poll', 'question_type', 'text']
-    raw_id_fields = ['poll']
+    list_display = ['id', 'poll', 'question_type', 'text', 'img']
+    raw_id_fields = ['poll', 'header']
     search_fields = ['question_type', 'text']
     inlines = [OptionsInline]
 
+    def img(self, instance):
+        header = instance.header
+        if not header or not header.image:
+            return 'n/a'
```

Comment:
why return n/a here instead of just `None`?

Reply:
@zoek1 could you fix this up in a follow up PR please ?